README.org

```diff
@@ -6,7 +6,7 @@ Most of the following packages are available in [[https://github.com/milkypostma
 
 #+BEGIN_QUOTE
 *Table of Contents*
-- [[#awesome-emacs][Awesome Emacs]]
+- [[#awesome-emacs-[[httpsgithubcomsindresorhusawesome][httpscdnrawgitcomsindresorhusawesomed7305f38d29fed78fa85652e3a63e154dd8e8829mediabadgesvg]]][Awesome Emacs [[https://github.com/sindresorhus/awesome][https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg]]]]
```

Comment:
I got it. Someone append a svg icon link just after the title (without newline) and `gen-toc` treat it as normal string.

And...Why delete PR....